nemo/export/tensorrt_llm.py

```diff
@@ -38,16 +38,24 @@
     is_nemo_file,
     load_nemo_model,
 )
-from nemo.export.trt_llm.qnemo import qnemo_to_tensorrt_llm
-from nemo.export.trt_llm.qnemo.tokenizer_utils import get_nmt_tokenizer
-from nemo.export.trt_llm.qnemo.utils import is_qnemo_checkpoint
 from nemo.export.trt_llm.tensorrt_llm_build import build_and_save_engine
 from nemo.export.trt_llm.tensorrt_llm_run import generate, generate_streaming, load, load_distributed, refit
 
+LOGGER = logging.getLogger("NeMo")
+
+use_model_opt = True
```

Comment:
I'll prepare MR to remove modelopt dependency in `nemo.export`

Reply:
Thanks Jan! You can remove that guard in your PR.